learning/tour-of-beam/frontend/lib/pages/tour/state.dart

```diff
@@ -71,6 +135,10 @@ class TourNotifier extends ChangeNotifier with PageStateMixin<void> {
       currentUnitContent?.solutionSnippetId != null;
   bool _isShowingSolution = false;
   bool get isShowingSolution => _isShowingSolution;
+  bool get isCurrentUnitCodeSaved =>
```

Comment:
```suggestion
  bool get doesUserSnippetExist =>
```

Reply:
`doesUserSnippetExist` doesn't contain `currentUnit`, but with it gets too clumsy.

Reply:
No.